app/views/documents/edit.html.erb

```diff
@@ -16,7 +16,7 @@
     gtm: "edit-document-submit",
   }) do |f| %>
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-grid-column-two-thirds" id="edit-document-title">
```

Comment:
Do these need to be as long as they are? It looks like we're now set up for pretty widespread collision prevention but I think there was only a couple of collisions.

Reply:
Doing some quick searching it looks like only the "body" id is used in places.

```
➜  content-publisher git:(attachment-upload) ✗ ag '#body'
spec/features/editing_images/upload_image_spec.rb
88:    expect(find("#body").value).to match snippet

spec/features/editing_content/insert_inline_image_spec.rb
40:    expect(find("#body").value).to match snippet

spec/features/editing_content/insert_video_embed_spec.rb
37:    expect(find("#body").value).to match snippet

spec/features/editing_content/insert_inline_file_attachment_spec.rb
60:    expect(find("#body").value).to match snippet
67:    expect(find("#body").value).to match snippet
```

We could just change the textarea id to be `#body-field` to make that id free for the anchor.

Reply:
Is there a specific issue with long ids? Or this just a standards thing?

Reply:
Just a question of why make them long if we don't need to - particularly as these will form part of a URL whereas most ids a user never sees so can be long and ugly without a user ever seeing.